pallets/subbridge/src/chainbridge.rs

```diff
@@ -997,13 +1016,14 @@ pub mod pallet {
 
 			// Notify relayer the crosschain transfer
 			let nonce = Pallet::<T>::bump_nonce(dest_id);
-			BridgeEvents::<T>::append(BridgeEvent::FungibleTransfer(
+			BridgeEvents::<T>::try_append(BridgeEvent::FungibleTransfer(
```

Comment:
If it fails at this point, the storage could be already mutated. So we must make sure the outer function is `#[transactional]` to avoid leaving dirty data in the storage.

Reply:
If the outside caller is `xtransfer.transfer()` which is `transactional`, but not sure the intrinsic transaction is `transactional` by default.

Reply:
Add `transactional` on all `BridgeTransactorImpl` functions